packages/store/src/transformer/job.ts

```diff
@@ -213,6 +213,25 @@ export class Job {
     return model;
   }
 
+  async snapshotToModelData(snapshot: BlockSnapshot) {
```

Comment:
Please keep using the arrow method like other methods.

Reply:
Most places use default method by default. But aligning to the nearby implementation surely makes more sense.